src/test/java/org/vafer/jdeb/changes/ChangesFileBuilderTestCase.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Marko.koren.
+ *
```

Comment:
Please use "The jdeb developers." for the copyright. I will add you as contributor before the next release.

Reply:
Forgot that netbeans adds it on new files. I'll remove it.